src/pages/blog/2024-10-08-graphql-conf-2024-wrap/index.mdx

```diff
@@ -0,0 +1,43 @@
+
+![Photo of the Unconference Topics board](./unconf.jpg)
+
+## Have You Heard the Word About Trusted Documents
+Data and application security are as relevant as ever. Our own plucky, GraphQL Community Gardener Benjie Gillam was on hand to tell attendees about one easy thing they can do to harden their GraphQL security posture and boost performance - use Trusted Documents! If you’re already using Trusted Documents, check out [Top 10 GraphQL Security Checks for Every Developer](https://youtu.be/fA81OFu9BVY).
+
+![Photo from a GraphQL workshop](./workshop.jpg)
+
+## Our Community had an Amazing Time 
+Between the excellent content and workshops, focused unconference sessions, Working Group member chats, TSC and Board meetings, and quality bond-building between attendees, we made the most out of our short time together. We can’t wait to do it again, and are so thankful for our sponsors and attendees for making it all happen. In the words of a few of our attendees:
+
+*“As a beginner in GraphQL, it was very helpful to see real use cases and honest accounts of the challenges along the way. I learned a lot about performance and security and had a great opportunity to network with other participants and potential vendors." - Nicolai Draslov, Danish Agency for Digital Government* 
+
+*"Great community. It's the first conference I've been to where multiple people remembered me from previous conferences."*
+
+*"GraphQL is evolving to new use cases every day and it's really a competitive advantage to experience them first hand with everyone that matters. I look forward the next edition!" - Vincent Desmares, Teamstarter*
+
+*"GraphQLConf 24 was a well organized event that empowered new and existing organizations to adopt GraphQL, navigating how to rollout GraphQL within their organizations by building understanding of eco-system." - Satish Chitnis* 
+
+## GraphQL Conf 2025 is coming to Europe!
+Lee Byron shared that the GraphQL Foundation Board is committed to hosting the next edition of GraphQL Conf somewhere in Europe in 2025. Planning is still in the early stages, so stay tuned, but this was welcome news for the dozens of community members who flew in from as far away as Belgium to join us in San Francisco.
+
+**Interested in getting involved? Drop a line to graphqlconf@graphql.org!**
+
+If you want to relive the experience of GraphQL Conf or just re-watch and share a favorite talk, you can check out all the [photos](https://www.flickr.com/photos/linuxfoundation/albums/72177720320386221/) and [videos](https://graphql.org/conf/2024/schedule/) on our conference website. Subscribe to our [YouTube](https://www.youtube.com/@GraphQLFoundation), [X](https://twitter.com/graphql), and [LinkedIn](https://www.linkedin.com/company/graphql-foundation/) channels to get content you may have missed and find out when we launch the new GraphQL Conf 2025 website!
```

Comment:
@jorydotcom Should this be the FoundationTalks YouTube rather than the Foundation one?

Reply:
I thought about it and kept changing my mind, because there's not a lot there yet.... but, I think what I'll actually do is add both!